api/MediaDevices.json

```
@@ -503,6 +503,546 @@
+            "tags": [
+              "web-features:web-audio"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "≤75"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "≤65"
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "return_object_property_channelCount": {
+          "__compat": {
+            "description": "`channelCount` property in returned object",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaTrackConstraints/channelCount",
+            "spec_url": "https://w3c.github.io/mediacapture-main/#dom-mediatrackconstraintset-channelcount",
+            "tags": [
+              "web-features:web-audio"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "≤75"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "≤65"
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "return_object_property_deviceId": {
+          "__compat": {
+            "description": "`deviceId` property in returned object",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaTrackConstraints/deviceId",
+            "spec_url": "https://w3c.github.io/mediacapture-main/#dom-mediatrackconstraintset-deviceid",
+            "tags": [
+              "web-features:web-audio"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "≤75"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "≤65"
```

https://github.com/caugner/firefox-releases/blame/097c91e6f23bd15b48423dd53583960353d8298d/dom/webidl/MediaTrackSupportedConstraints.webidl#L25

Note: Added in 42, but method itself only added in 44.

```suggestion
                "version_added": "44"
```